<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-GE6LMT5" failures="1" tests="1" name="Test" time="50.252" errors="0" timestamp="2025-06-24T22:04:31 IST">
  <testcase classname="Tests.HotstarPlayerScreenOutputs" name="jioHotsterFailedTestCase" time="4.403">
    <failure type="java.lang.AssertionError" message="Intentionally failing the testcase expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: Intentionally failing the testcase expected [false] but found [true]
at Tests.HotstarPlayerScreenOutputs.jioHotsterFailedTestCase(HotstarPlayerScreenOutputs.java:90)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- jioHotsterFailedTestCase -->
  <testcase classname="Tests.HotstarPlayerScreenOutputs" name="jioHotstarPlayreResult" time="0.0">
    <ignored/>
  </testcase> <!-- jioHotstarPlayreResult -->
</testsuite> <!-- Test -->
